cfme/tests/containers/test_cockpit.py

```diff
@@ -12,7 +12,7 @@
 
 
 @pytest.mark.polarion('CMP-10255')
-@pytest.mark.meta(blockers=[BZ(1406772, forced_streams=["5.7", "5.8"])])
+@pytest.mark.meta(blockers=[BZ(1406772, forced_streams=["5.8", "5.9"])])
```

Comment:
**REQUIRED**: This bugzilla is `CLOSED INSUFFICIENT_DATA`. Please remove this blocker entry.